transform/mattermost-analytics/models/marts/product/_product__models.yml

```diff
@@ -240,3 +217,19 @@ models:
       - name: is_missing_activity_data
         description: Whether server activity data are available for the given date and server id.
 
+  - name: dim_server_info
+    description: Static information for a given server.
+    columns:
+      - name: server_id
+        description: The server's unique id.
+        tests:
+          - not_null
+      - name: installation_id
+        description: The id of the installation. Exists only for cloud installations.
+      - name: hosting_type
+        description: |
+          Type of hosting for this server. Can be one of:
+          - `Cloud`: server always reports an installation id
+          - `Self-hosted`: server always reports a null installation id.
+          - `Probably cloud`: server reports an installation id more often than not reporting as installation id.
```

Comment:
IMO, we shouldn't have these 2 text (`Probably cloud` and `Probably self-hosted` in the data, its speculative and might leave the end user confused, until we surely know if its `Cloud` or `Self-Hosted`, which we are going to in the near future, combining other sources of data, we should leave this blank.

Reply:
Thank you for the feedback. Should `Probably cloud`, `Probably self-hosted` and null values be replaced with something like `Unknown`? We can also use the number of rows with that value as a quality metric.